src/toProgramUpdateEvent.ts

```diff
@@ -0,0 +1,28 @@
+type ProgramUpdateEvent = {
+  programId: string;
+};
+
+const isProgramUpdateEvent = (data: unknown): data is ProgramUpdateEvent => {
+  if (typeof data === "object") {
```

Comment:
it's better coding style to write if's without else this can be :
```
if (typeof data !=== "object) { 
   return false;
}
```
same for if (data) 

you will notice it's easier to read

Reply:
there are few if statements in this PR that can be changed but if you like if else and alot of branched curly braces I won't judge